src/test/java/com/restassured/test/GetAndPostExample.java

```diff
@@ -8,34 +9,28 @@
 import static io.restassured.RestAssured.*;
 import static org.hamcrest.Matchers.*;
 
-import java.util.HashMap;
-import java.util.Map;
-
 public class GetAndPostExample {
 	
 	@Test
 	public void testGet() {
-		
-		baseURI =  "https://reqres.in/api";
+		RestAssured.baseURI = "https://reqres.in/api";
```

Comment:
I have the question regarding `RestAssured.baseURI`
Why you believe is better to define the RestAssured class instead of just use baseURI as default (with the import `import static io.restassured.RestAssured.*;`)
Here and all other parts where you change to `RestAssured.baseURI`